docs/source/records_mover.records.rst

```diff
@@ -0,0 +1,82 @@
+records\_mover.records package
+==============================
+
+Subpackages
+-----------
+
+.. toctree::
+   :maxdepth: 4
+
+   records_mover.records.sources
+   records_mover.records.targets
```

Comment:
Most of the 'documenting only the public API' comes from judicious exclusion of things from these subpackage lists.